src/commands/firestore-indexes-list.ts

```diff
@@ -13,13 +13,18 @@ export const command = new Command("firestore:indexes")
     "Pretty print. When not specified the indexes are printed in the " +
       "JSON specification format."
   )
+  .option(
+    "--database <databaseId>",
+    "Database ID of the firestore database from which to list indexes. (default) if none provided."
+  )
   .before(requirePermissions, ["datastore.indexes.list"])
   .before(warnEmulatorNotSupported, Emulators.FIRESTORE)
   .action(async (options: any) => {
```

Comment:
Optional: While you're making changes to these commands, it would be nice to give options a type other than 'any' - this would silence the many unsafe assignment warnings  throughout these commands. 

We have a base type in  [options.ts](https://github.com/firebase/firebase-tools/blob/master/src/options.ts)  that contains the fields that are present in all commands. Ideally, you'd do something like hosting does in [hosting/options.ts](https://github.com/firebase/firebase-tools/blob/master/src/hosting/options.ts), where we extend the base type with fields specific to the command, and then use that.

Reply:
good idea - done!